laser_encoders/test_laser_tokenizer.py

```diff
@@ -35,6 +39,27 @@ def input_text() -> str:
     return "This is a test sentence."
 
 
+@pytest.fixture
+def test_readme_params() -> dict:
+    return {
+        "lang": "igbo",
+        "input_sentences": ["nnọọ, kedu ka ị mere"],
+        "expected_embedding_shape": (1, 1024),
+        "expected_array": [
+            0.3807628,
+            -0.27941525,
+            -0.17819545,
+            0.44144684,
+            -0.38985375,
+            0.04719935,
+            0.20238206,
+            -0.03934783,
+            0.0118901,
+            0.28986093,
```

Comment:
Just FYI due to the update from Siddharth, on Sacremoses preprocessing these numbers will likely change.

Reply:
Let's fix them in the PR where we update the interaction with Sacremoses!